src/halfpipe/cluster.py

```
@@ -122,7 +122,7 @@ def make_script(workdir: Path, graphs: Dict[str, Any], opts):
 
     data = dict(
         n_chunks=n_chunks,  # one-based indexing
-        singularity_container=os.environ["SINGULARITY_CONTAINER"],
+        singularity_container=os.environ.get("SINGULARITY_CONTAINER", "/containers/halfpipe-halfpipe-latest.sif"),
```

I would prefer to not hard-code a default here. Maybe we can raise an error?